lib/Basic/Unix/TaskQueue.inc

```diff
@@ -107,9 +107,11 @@ public:
   /// \returns true on error, false on success
   bool execute();
 
-  /// \brief Reads data from the pipes, if any is available.
-  /// \returns true on error, false on success
-  bool readFromPipes();
+  /// \brief Reads data from the pipes, if any is available. If
+  /// \p untilEnd is true, reads until the end of the stream; otherwise
+  /// reads once (posslby with a retry on EINTR), and returns.
+  /// \returns true on error, false on success.
+  bool readFromPipes(bool untilEnd=true);
```

Comment:
I hadn't seen that formatting of = without spaces before. Does git-clang-format let it pass?

Reply:
No, I did not clang-format this.

Reply:
I think convention definitely skews towards including the spaces.

Reply:
fixed